### ISSUE TITLE
Bayesian Optimization to find optimal hyperparameters

### ISSUE THREAD

**Issue by andrewjong** [andrewjong/Estrous-AI#13]
Learning rate, optimizer, etc.

https://github.com/fmfn/BayesianOptimization

(Suggested by Toni. Need to ask him more about how to do this)

**Comment by andrewjong** [comment]
Implemented
